add-ons/prometheus/README.md

```diff
@@ -15,13 +18,17 @@ minishift addon apply prometheus --addon-env namespace=kube-system
 _NOTE_: You should provide the namespace where it will be installed with the addon-env namespace, like this:
 
 ## Use prometheus
-Prometheus will be available at:
 
-```
-$ minishift openshift service prometheus -n <namespace>
-```
+- Prometheus will be available at:
+
+  ```
+  $ minishift openshift service prometheus -n <namespace>
+  ```
+
+  _NOTE_: There service is exposed though SSL, so use _https_ to access it.
+
+- Login with user  'admin' and pasword 'admin' in OpenShift console during `Sign in with a Account` action.
```

Comment:
I would say a "cluster-admin" user as the user might have created an admin user in any way. And then, maybe point to the admin-user addon in case he don't know how to enable it.